src/toxcore/tcp/server/client.rs

```diff
@@ -77,6 +90,38 @@ impl Client {
         self.ping_id
     }
 
+    /** Last time PingRequest sent.
+    */
+    pub fn last_pinged(&self) -> Instant { self.last_pinged }
+
+    /** Set last_pinged
+    */
+    pub fn set_last_pinged(&mut self, time: Instant) {
```

Comment:
No need in these two methods, since we have now more abstract `is_pong_timedout` and `is_ping_interval_passed`.

Reply:
done